.github/workflows/triage-issues.yml

```diff
@@ -0,0 +1,48 @@
+name: Triage issues
```

Comment:
Maybe a comment at the top of this file:

```
# This file is synced from the `.github` repository, do not modify it directly.
```

Reply:
Good idea 👍